src/main/java/jewellery/inventory/service/UserInOrganizationService.java

```diff
@@ -61,15 +60,14 @@ public OrganizationSingleMemberResponseDto updateUserPermissionsInOrganization(
   @LogCreateEvent(eventType = EventType.ORGANIZATION_USER_CREATE)
   public OrganizationSingleMemberResponseDto addUserInOrganization(
       UUID organizationId, UserInOrganizationRequestDto userInOrganizationRequestDto) {
-
-    organizationService.validateCurrentUserPermission(
-        organizationService.getOrganization(organizationId), OrganizationPermission.MANAGE_USERS);
-
     Organization organization = organizationService.getOrganization(organizationId);
 
     organizationService.validateCurrentUserPermission(
         organization, OrganizationPermission.MANAGE_USERS);
 
+    validateUserIsPartOfOrganization(
```

Comment:
We are actually validating that the user is not part of the organization:
```suggestion
    validateUserIsNotPartOfOrganization(
```